rstudio/c9s-python-3.9/Dockerfile

```diff
@@ -45,6 +45,11 @@ RUN chmod 1777 /var/run/rstudio-server && \
     mkdir -p /usr/share/doc/R
 COPY rsession.conf /etc/rstudio/rsession.conf
 
+# package installation 
+RUN  dnf install -y libsodium-devel.x86_64 && dnf install -y libgit2-devel.x86_64 && dnf install -y libcurl-devel && dnf install -y cmake && dnf install -y flexiblas-*
+RUN R -e 'install.packages(c("tidyverse", "tidymodels", "plumber", "vetiver", "devtools"))'
```

Comment:
These packages are not installed on the image, could you please investigate? 
It is likely that you need to verify the absence of any required additional OS packages.
(for your reference: https://stackoverflow.com/questions/48469005/install-devtools-package-in-ubuntu-vm)